### PR TITLE
Feature/create pop ups

### PR DESCRIPTION
**Pull request by AnaIoramashvili**

### CODE REVIEW

File: QuizApp/Custom/CustomPopUp.swift [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        stackView.spacing = Constants.Sizing.stackViewSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+    
+    private let questionLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.numberOfLines = .zero
+        label.font = .systemFont(ofSize: Constants.FontSize.medium, weight: .bold)
```
Comment (tamokakhidze) [inline QuizApp/Custom/CustomPopUp.swift:25]:
```suggestion
        label.font = .systemFont(
        ofSize: Constants.FontSize.medium, 
        weight: .bold
        )
```

File: QuizApp/Custom/CustomPopUp.swift [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        stackView.alignment = .center
+        stackView.spacing = Constants.Sizing.buttonsSpacing
+        stackView.distribution = .fillEqually
+        return stackView
+    }()
+
+    lazy var rejectButton: PopupButton = {
+        let button = PopupButton()
+        button.configure(title: Constants.Texts.rejectButtonTitle)
+        button.addTarget(self, action: #selector(handleNoButtonTap), for: .touchUpInside)
```
Comment (tamokakhidze) [inline QuizApp/Custom/CustomPopUp.swift:41]:
```suggestion
        button.addTarget(
        self, 
        action: #selector(handleNoButtonTap), 
        for: .touchUpInside
        )
```

**Review by tamokakhidze — COMMENTED** [review]
Few suggestions. But it is totally up to you

File: QuizApp/Custom/CustomPopUp.swift [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            rejectButton,
+            acceptButton
+        )
+        stackView.addArrangedSubviews(
+            questionLabel,
+            buttonsStackView
+        )
+    }
+    // MARK: - Constraints
+    private func setConstraints() {
```
Comment (LukaGujejiani) [inline QuizApp/Custom/CustomPopUp.swift]: Function is just calling another function and nothing more

**Review by LukaGujejiani — CHANGES_REQUESTED** [review]

File: QuizApp/Custom/CustomPopUp.swift (hunk 1 above)
Comment (AnaIoramashvili) [reply QuizApp/Custom/CustomPopUp.swift:25]: I prefer label.font = .systemFont(ofSize: Constants.FontSize.medium, weight: .bold) this styling

File: QuizApp/Custom/CustomPopUp.swift (hunk 2 above)
Comment (AniDzamelashvili) [reply QuizApp/Custom/CustomPopUp.swift:41]: it is marked as resolved but suggestion is still not applied

**Review by AniDzamelashvili — APPROVED** [review]

**Review by LukaGujejiani — APPROVED** [review]

**Review by tamokakhidze — APPROVED** [review]

**Pull request merged**